tasmota/tasmota_xdrv_driver/xdrv_85_esp32_ble_eq3_trv.ino

```diff
@@ -480,16 +480,18 @@ int EQ3ParseOp(BLE_ESP32::generic_sensor_t *op, bool success, int retries){
 
     ResponseAppend_P(PSTR(",\"hassmode\":"));
     do {
+      //HASS allowed modes [“auto”, “off”, “cool”, “heat”, “dry”, “fan_only”]
       //0201283B042A
-      // its in auto
+      // If its in auto or holiday, set to auto
       if ((stat & 3) == 0) { ResponseAppend_P(PSTR("\"auto\"")); break; }
-      // it's set to 'OFF'
+      // If its in manual and 4.5°C, set to off
       if (((stat & 3) == 1) && (status[5] == 9)) { ResponseAppend_P(PSTR("\"off\"")); break; }
-      // it's actively heating (valve open)
+      // If its in manual above 4.5°C and valve is open, set to heat
       if (((stat & 3) == 1) && (status[5] > 9) && (status[3] > 0)) { ResponseAppend_P(PSTR("\"heat\"")); break; }
-      // it's achieved temp (valve closed)
-      if (((stat & 3) == 1) && (status[5] > 9)) { ResponseAppend_P(PSTR("\"idle\"")); break; }
-      ResponseAppend_P(PSTR("\"idle\""));
+      // If its in manual above 4.5°C and valve is closed, set to off
+      if (((stat & 3) == 1) && (status[5] > 9) && (status[3] > 0)) { ResponseAppend_P(PSTR("\"off\"")); break; }
```

Comment:
Did an oopsie here, found in testing. will change to 
if (((stat & 3) == 1) && (status[5] > 9) && (status[3] == 0)) { ResponseAppend_P(PSTR("\"off\"")); break; }